src/api/services/jobs.ts

```diff
@@ -39,24 +39,52 @@ type JobExpiryMap = {
  * Each job type must define an input schema and may optionally define a result schema.
  */
 export const jobTypeSchemas: JobSchemaMap = {
-  CRITERIA_POLYGONS: {
+  TEST: {
     input: z
       .object({
-        // TODO actual payload
+        // This is just to break up the hash
         id: z.number(),
       })
       .strict(),
+    result: z.object({}).strict().optional(),
+  },
+  // The suitability assessment job takes in regional parameters and returns the location of the file relative to the job storage location.
+  SUITABILITY_ASSESSMENT: {
+    input: z
+      .object({
+        // High level config
+        region: z.string().describe('Region for assessment'),
+        reef_type: z.string().describe('The type of reef, slopes or flats'),
+        // Criteria
+        depth_min: z.number().describe('The depth range (min)'),
+        depth_max: z.number().describe('The depth range (max)'),
+        slope_min: z.number().describe('The slope range (min)'),
+        slope_max: z.number().describe('The slope range (max)'),
+        x_dist: z.number().describe('Length (m) of the target polygon'),
+        y_dist: z.number().describe('Width (m) of the target polygon'),
+        rugosity_min: z.number().describe('The rugosity range (min)'),
+        rugosity_max: z.number().describe('The rugosity range (max)'),
+        threshold: z.number().describe('Suitability threshold integer (min)'),
```

Comment:
don't need to change now, but may be better to just make criteria generic (slope, depth, rugosity, etc), putting them in separate criteria dict would be cleaner